### PR TITLE
First CodingBat Assignment

### PR DESCRIPTION
**Pull request by bdferr**
This is my first ever assignment from CodingBat.com! A winner is me. You can view the working post here: http://bdferr.github.io/spring2014/2014/02/07/bdferrcodingbat1.html

### CODE REVIEW

**Comment by eah13** [comment]
@bdferr nice job working ahead, but I still need you to post 3 problems from the -1 problem sets with you code to them.  We'll go over sum67 next week.

**Comment by bdferr** [comment]
Alright; I have just fixed that.

Thanks,

Brendan

File: _posts/2014-01-31-stringsandlists.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 author: bdferr
-title: strings_and_lists
+title: Bdferr Strings and Lists
```
Comment (eah13) [inline _posts/2014-01-31-stringsandlists.md:4]: Thx for fixing this!

**Comment by eah13** [comment]
Well done on your updated reflections.  You seem to really be enjoying yourself!

**Pull request merged**